tests/integration/kubernetes/gha-run.sh

```diff
@@ -241,6 +241,56 @@ install_kata_tools_placeholder() {
 		"after CI picks up the gha yaml changes required to test that installation."
 }
 
+function deploy_snapshotter() {
+	echo "::group::Deploying ${SNAPSHOTTER}"
+
+	case ${SNAPSHOTTER} in
+		nydus) deploy_nydus_snapshotter ;;
+		*) >&2 echo "${SNAPSHOTTER} flavour is not supported"; exit 2 ;;
+	esac
+
+	echo "::endgroup::"
+}
+
+function deploy_nydus_snapshotter() {
+    echo "::group::deploy_nydus_snapshotter"
+	ensure_yq
+
+	local nydus_snapshotter_install_dir="/tmp/nydus-snapshotter"
+	if [ -d "${nydus_snapshotter_install_dir}" ]; then
+		rm -rf "${nydus_snapshotter_install_dir}"
+	fi
+	mkdir -p "${nydus_snapshotter_install_dir}"
+	nydus_snapshotter_version=$(get_from_kata_deps "externals.nydus-snapshotter.version")
+	git clone -b "${nydus_snapshotter_version}" https://github.com/containerd/nydus-snapshotter "${nydus_snapshotter_install_dir}"
+
+	pushd "$nydus_snapshotter_install_dir"
+	if [ "${PULL_TYPE}" == "guest-pull" ]; then
+		# Enable guest pull feature in nydus snapshotter
+		yq write -i misc/snapshotter/nydus-snapshotter.yaml 'data.FS_DRIVER' "proxy" --style=double
+	else
+		>&2 echo "Invalid pull type"; exit 2 
+	fi
+	
+	# Disable to read snapshotter config from configmap
+	yq write -i misc/snapshotter/nydus-snapshotter.yaml 'data.ENABLE_CONFIG_FROM_VOLUME' "false" --style=double
+	# Enable to run snapshotter as a systemd service
+	yq write -i misc/snapshotter/nydus-snapshotter.yaml 'data.ENABLE_SYSTEMD_SERVICE' "true" --style=double
+	# Enable "runtime specific snapshotter" feature in containerd when configuring containerd for snapshotter
+	yq write -i misc/snapshotter/nydus-snapshotter.yaml 'data.ENABLE_RUNTIME_SPECIFIC_SNAPSHOTTER' "true" --style=double
+
+	# Deploy nydus snapshotter as a daemonset
+	kubectl create -f "nydus-snapshotter-rbac.yaml"
```

Comment:
Wrong path to yamls, should have `misc/snapshotter/`:
```
kubectl create -f misc/snapshotter/nydus-snapshotter-rbac.yaml
kubectl create -f misc/snapshotter/nydus-snapshotter.yaml
```

Reply:
Fixed.